### PR TITLE
Parse osc xml output to find only the relevant build statuses

### PR DESCRIPTION
**Pull request by jimmykarily**
Signed-off-by: Christian Richter <crichter@suse.com>

### CODE REVIEW

File: lib/cf_obs_binary_builder/obs_package.rb [hunk 1]
```diff
@@ -1,7 +1,13 @@
+require "nokogiri"
 class CfObsBinaryBuilder::ObsPackage
   attr_reader :name, :obs_project
 
   ARTIFACT_EXTENSION_REGEXP=".tgz|.tar.gz|.tbz|tar.bz|.zip|.phar"
+  STACK_TO_OBS_REPOSITORY = {
+    "sle12" => "SLE_12_SP3",
+    "sle15" => "SLE_15",
```
Comment (thardeck) [inline lib/cf_obs_binary_builder/obs_package.rb]: The stack is called `cfsle15fs`

File: spec/cf_obs_binary_builder/obs_package_spec.rb [hunk 2]
```diff
@@ -10,29 +10,70 @@
...
+      <status package="bundler-1.16.2" code="succeeded" />
+    </result>
+    <result  project="Cloud:Platform:buildpacks:dependencies" repository="IRRELEVANT_STACK" arch="x86_64" code="published" state="published">
+      <status package="bundler-1.16.2" code="failed" />
+    </result>
+  </resultlist>
 EOF
+  end
+
+  let(:stacks){ ["sle12","sle15"] }
```
Comment (thardeck) [inline spec/cf_obs_binary_builder/obs_package_spec.rb]: same as above

File: lib/cf_obs_binary_builder/obs_package.rb (hunk 1 above)
Comment (dragonchaser) [reply lib/cf_obs_binary_builder/obs_package.rb]: done

File: spec/cf_obs_binary_builder/obs_package_spec.rb (hunk 2 above)
Comment (dragonchaser) [reply spec/cf_obs_binary_builder/obs_package_spec.rb]: done

**Comment by thardeck** [comment]
@dragonchaser it woudl be nice if you could also fix the sle15 references in the readme and comments from the last pr.

File: lib/cf_obs_binary_builder/obs_package.rb (hunk 1 above)
Comment (mudler) [reply lib/cf_obs_binary_builder/obs_package.rb]: might be relevang also for: https://github.com/SUSE/cf-obs-binary-builder/commit/69f75a171b6876c3559c964a500e8f255a7aa5d7#diff-c6acfca63885a53f914d8ee2530e6680R54  and https://github.com/SUSE/cf-obs-binary-builder/commit/f21e604b1af2aa29eceb39099710e76a89dfc3f1

**Pull request merged**